### PR TITLE
Primary section comparator

### PR DESCRIPTION
**Pull request by katsuraku**

### CODE REVIEW

**Comment by jonathan-best** [comment]
Do you have any examples of articles that didn't have a section/primary section?

**Comment by jonathan-best** [comment]
:sunglasses:

**Comment by katsuraku** [comment]
Unit tests added

File: test/components/sectionModifier.spec.js [hunk 1]
```diff
@@ -10,10 +10,12 @@ import TimespanSelector from '../../src/shared/components/TimespanSelector';
 
 describe ('Modifier component', function() {
   let modifier;
+  let modifier2;
```
Comment (jonathan-best) [inline test/components/sectionModifier.spec.js]: What are these for?

**Comment by jonathan-best** [comment]
:sunglasses:

File: src/server/esClient.js [hunk 2]
```diff
@@ -131,6 +132,31 @@ export function getMetaData (uuid) {
...
 
+// Normalise the various primary section formats we can get
+export function _normalisePrimarySection(primarySections) {
+  if (Array.isArray(primarySections)) {
+    return primarySections[0];
+  }
+  else if (typeof(primarySections) === 'string') {
+    return primarySections.split(',')[0];
+  }
+}
```
Comment (StuartFT) [inline src/server/esClient.js]: hmm @Lucas-Gluchowski @ookejimift seems like we should do this processing in logstash if it's necessary?

**Pull request merged**